WebGUI/html/FEMacroTest.html

```diff
@@ -280,6 +280,25 @@
 			{
 				firstTime = false;
 			}
+			var windowTooltip = "Welcome to the FE Macros web app!\n\n" + 
+				"This app provides an interface for devices configured in the context. " +
+				"It allows for common functions such as resets and reading and writing to memory. . " +
+				"\n\n" +
+				"Choose a FE type (e.g., CFO, DTC, or ROC) to access functionalities. " +
```

Comment:
otsdaq_utilities is experiment agnostic; so we should put any experiment specific terminology.

Instead of "(e.g. CFO, DTC, or ROC)" I think we should say (i.e. this should match your FE plugin class name)